applications/StructuralMechanicsApplication/CMakeLists.txt

```diff
@@ -179,39 +39,33 @@ pybind11_add_module(KratosStructuralMechanicsApplication MODULE THIN_LTO ${KRATO
 target_link_libraries(KratosStructuralMechanicsApplication PUBLIC KratosStructuralMechanicsCore)
```

Comment:
@roigcarlo I meant this one, should it be private or public?

Fluid does it private